python/darpinstances/instance.py

```diff
@@ -195,6 +199,15 @@ def map_equipment_type(equipment_str: str) -> EquipmentType:
     }
     return equipment_mapping.get(equipment_str, EquipmentType.NONE)
 
+def time_str_to_seconds (date_str: str) -> int:
```

Comment:
This will fail if the operation time spans multiple days (night shifts). It should use the Python datetime handling (datetime.strptime...)